lib/jekyll-admin/server/static_file.rb

```diff
@@ -47,7 +47,7 @@ def static_file_body
       end
 
       def static_files
-        site.static_files
+        JekyllAdmin::StaticDirectory.new(site.source, base_directory: site.source)
```

Comment:
Shouldn't the path, by default, be `/`? I'd think we'd want to be passing the *relative* path, not the absolute path, if we already have the base from the second argument.